python/requirements-optional.txt

```diff
@@ -1,5 +1,5 @@
 hidapi>=0.7.99.post20
 web3>=5
-Pillow
+Pillow>=10
```

Comment:
this is also missing from setup.py, see 3c22a09f9d8c67ac68490f69925a0d471122d874

Reply:
i mean, it might still make sense to merge this separately from GFL which is presumably going to take longer to land